validator/src/main/java/com/google/daq/mqtt/registrar/Registrar.java

```diff
@@ -185,6 +190,9 @@ public class Registrar {
   private boolean doNotUpdate;
   private boolean expandDependencies;
   private boolean updateMetadata;
+  private String currentRunTimestamp;
```

Comment:
nit: I'd either make both these `Instant` or both `String` (iso) -- seems weird to have them different.  Just weird, not necessarily bad so if it doesn't make sense then don't worry about it.